portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,47 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  padding: 5px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+body {background-color: rgb(196, 238, 243);
+      font-family: 'Courier New';}
+h1   {color: #ff9999; font-family: Tahoma, Geneva, sans-serif;}
+p    {color: #b30000;}
+hr   {height:1px; background-color:#ff9999;}
+
+button {
+  background-color: rgb(104, 155, 196);
+  border: none;
+  color: white;
+  padding: 8px 20px;
+  text-align: center;
+  display: inline-block;
+  font-size: 13px;
+  cursor: pointer;
+  font-family: Tahoma, Geneva, sans-serif;
+}
+
+input[type="text"] {
```

Comment:
Shouldn't you define a class for this?

Reply:
I saw examples of this online, so it seems it's a common practice

Reply:
The problem with this approach is that if you have a big website with multiple input text boxes, all of them will be styled according to this CSS rule. I think that usually having a specific rule for each element (or element group) is a better practice. 

You can leave as-is for the starter projects, in any case.